### PR TITLE
Bump line length to 88

### PR DESCRIPTION
**Pull request by Lunarmagpie**
It improves clarity over 79
note this PR 100% has to be merged with main before it is merged

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,11 +26,11 @@ isort = "^5.10.1"
 mypy = "^0.991"
 
 [tool.black]
-line-length = 79
+line-length = 88
 skip-magic-trailing-comma = true
 
 [tool.ruff]
-line-length = 79
+line-length = 88
```
Comment (CircuitSacul) [inline pyproject.toml:33]: both ruff and black use 88 by default, I think you can just remove the config

**Review by CircuitSacul — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (Lunarmagpie) [reply pyproject.toml:33]: done

**Review by trag1c — APPROVED** [review]

**Review by MithicSpirit — CHANGES_REQUESTED** [review]
Disapprove. 80 maximum.

**Review by CircuitSacul — CHANGES_REQUESTED** [review]
Convince Mithic first.

**Review by CircuitSacul — APPROVED** [review]

**Review by trag1c — APPROVED** [review]

**Review by Endercheif — APPROVED** [review]
:ball:

**Pull request merged**